scripts/linters/custom_eslint_checks/rules/match-line-break.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that  the line breaks between the
+ * dependencies listed in the controller of a directive or service exactly
+ * match those between the arguments of the controller function.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: (
+        'Lint check to ensure that  the line breaks between the dependencies' +
+        ' listed in the controller of a directive or service exactly' +
+        'match those between the arguments of the controller function'),
+      category: 'Stylistic Issues',
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      matchLineBreak: (
+        'Please ensure that the line breaks pattern between the dependencies' +
+        ' mentioned as strings and the dependencies mentioned as function' +
+        ' parameters for the corresponding controller should exactly match.')
+    },
+  },
+
+  create: function(context) {
+    var getDepLiteralLines = function(controllerArg, nameIn) {
+      var con = {};
+      var startLine = 1000000;
+      controllerArg.forEach(function(Literal) {
+        var lineNo = Literal.loc.start.line;
+        if (startLine > lineNo) {
+          startLine = lineNo;
+        }
+        con[Literal[nameIn]] = lineNo - startLine;
+      });
+      return con;
+    };
+    return {
+      'CallExpression[callee.property.name=directive]': function(node) {
+        var arg = node.arguments;
+        // In angular, components function take 2 arguments and type of last
+        // arguments is an ArrayExpression.
```

Comment:
Rephrase the comment something around this: `Checking if .........`. Also, add a small example as well.

Reply:
@AdityaDubey0 In my previous comment I suggested you to rephrase the comment but I gave the wrong example. As you can see, that example is saying 'what' this 'if' statement is doing, instead we should write comments that shows 'why' we are doing something. The if statement is clear here but why we are doing such a check is not clear. Hence add a comment so that it explains why we are adding this 'if' statement.

Reply:
Done!